setup.py

```diff
@@ -1,19 +1,12 @@
+#! /usr/bin/env python
 # -*- coding: utf-8 -*-
+"""Distribution setup module."""
 
-from setuptools import find_packages, setup
+__requires__ = ("setuptools >=39.0",)
```

Comment:
You could also use `pyproject.toml`'s `[build-system]` section (PEP 518/517). But be aware that it doesn't support editable installs often used for development.